specs/unified_kernel_image.md

```diff
@@ -55,7 +55,16 @@ UKIs consist of the following resources:
 * Optionally, a JSON file encoding expected PCR 11 hash values seen from userspace once the UKI has booted up, along with signatures of these expected PCR 11 hash values, in the `.pcrsig` section. The signatures must also match the abovementioned key pair.
 * Optionally, a CSV file encoding the SBAT metadata for the image, in the `.sbat` section. The [SBAT format is defined by the Shim project](https://github.com/rhboot/shim/blob/main/SBAT.md), and used for UEFI revocation purposes.
 
-Note that all of the sections defined above are singletons: they may appear once at most – except for the `.dtb` section which may be appear more than once.
+Note that all of the sections defined above are singletons:
+they may appear at most once,
+except for the `.dtb` section which may be appear many times.
+
+Only the `.linux` section is required for the image to be considered a Unified *Kernel* Image.
```

Comment:
maybe say explicitly here that boot menus such as sd-boot might have stricter requirements, i.e. require .osrel or so. But be vague here, I think that's something we should document in sd-boot documents, not here in these docs